### ISSUE TITLE
alert() title is "Message from webpage" (Windows)

### ISSUE THREAD

**Issue by pomplesiegel** [r0x0r/pywebview#150]
On Windows using WinForms, the title of the popup windows created by the javascript calls `alert()`, 'confirm()' and 'prompt()' is "Message from webpage", which can be a bit strange when deploying a desktop app. 

![image](https://user-images.githubusercontent.com/1450352/32297577-f2ea3704-bf25-11e7-918b-18dadada8785.png)

Hopefully the user never knows there's a webview front-end, but this makes that fact quite apparent. Any way to customize or change this? Even a blank title would be sufficient / better than the current title.

**Comment by shivaprsdv** [comment]
I have no idea. You cannot normally change the title of browser-implemented
pop ups. Only if _we_ had implemented it, like the alert in Cocoa, could we set
the title.

That said, I suppose you could try [this](https://stackoverflow.com/a/1905335/7908996). It says works in IE, might work here as
well.

**Comment by r0x0r** [comment]
@pomplesiegel, Would you try this VBScript hack and see if it works? If so, maybe it could be incorporated into pywebview by overriding window.alert.

**Comment by pomplesiegel** [comment]
Sure, i'll give it a shot. Thanks!

**Comment by pomplesiegel** [comment]
Unfortunately the VBScript hack did not work on my machine. I'm running Windows 10 with IE11. I believe IE11 is the first version where VBScript cannot be run. I tried a few tweaks like 
```html
<meta http-equiv="x-ua-compatible" content="IE=10">
```
as well, but to no avail. 

Any thoughts?

**Comment by r0x0r** [comment]
There is a browser emulation key in the registry that lets IE act as a specific version. pywebview automatically sets it to use the latest version of IE. It is possible to emulate an older IE version, but then again it is something we don't probably want. 

Another solution would be to construct a custom message box, but it would require a Javascript bridge described in #83. I have been investigating it on Windows, but it looks like it is impossible to implement it purely in Python.

**Comment by pomplesiegel** [comment]
Yes, this all makes sense, given that the "Message from a webpage" was quite intentional on IE's part, to make it clear to people what was going on. Well, not totally crucial! Thank you

**Comment by r0x0r** [comment]
I got the JS bridge working on Windows, so a custom message box is
definitely doable. Will update on this issue.

**Comment by pomplesiegel** [comment]
Great!

**Comment by r0x0r** [comment]
I got the basic implementation working in the `alert-box` branch. Will merge the branch after #83 is finished.

**Comment by shivaprsdv** [comment]
Now that #166 is merged, we should get this done.

**Comment by r0x0r** [comment]
Pushed an implementation to master. @pomplesiegel please test and close this issue if everything's ok.

**Comment by shivaprsdv** [comment]
If the working code is on `master`, the `alert-box` branch is no longer needed.

**Comment by r0x0r** [comment]
Yes, it was a hassle to rebase it, so I opted for a fresh start. The branch
can be deleted now.

**Comment by shivaprsdv** [comment]
Done. 👍
